### DISCUSSION TITLE
two way data binding

### DISCUSSION THREAD

**Discussion by a0m0rajab** [nuxt-community/firebase-module#583] [Q&A] [answered]
Is there any information about two-way data binding between the APP and firebase? 

I could find some information about vuefire but not about nuxt-firebase: 

https://vuefire.vuejs.org/vuefire/binding-subscriptions.html#programmatic-binding

it would be great to know if there is a way to start or any articles that covered the topic.

**Comment by lupas** [comment, marked as answer]
@a0m0rajab 
It's a bit outdated, but this example I once wrote shows the use of vuexfire in comination with nuxt/firebase:
https://firebase.nuxtjs.org/tutorials/vuexfire

1) How you would set up the store:
https://github.com/lupas/nuxt-fire-vuexfire-example/blob/master/store/index.js

2) How you would bind it on client- & server-side.
https://github.com/lupas/nuxt-fire-vuexfire-example/blob/master/pages/ssr-vuexfire/index.vue

Vuefire should work in a similar fashion. Hope that helps,

**Reply by a0m0rajab** [reply]
@lupas is it outdated because of the vue 3? or is there something else that have replaced it? 

Thank you for the answer, I will check those examples it will be helpful.

**Reply by lupas** [reply]
@a0m0rajab 
I just haven't had a look at it for a while. It for sure won't work with Firebase 9 anymore, but with 8 and before it should be fine I think :)
